src/sentry/api/endpoints/project_group_index.py

```diff
@@ -159,16 +159,23 @@ class GroupValidator(serializers.Serializer):
     ignoreUserCount = serializers.IntegerField()
     # in minutes, max of one week
     ignoreUserWindow = serializers.IntegerField(max_value=7 * 24 * 60)
-    assignedTo = UserField()
+    assignedTo = ActorField()
```

Comment:
this risk in merging this is that values being sent today won't work with this new field. so i need to make sure i cover all the cases (id, email, username, etc)